internal/models/stream_change.go

```diff
@@ -10,7 +10,7 @@ type ChannelUpdateEventSubEvent struct {
 	StreamLanguage       string `json:"language"`
 	StreamCategoryID     string `json:"category_id"`
 	StreamCategoryName   string `json:"category_name"`
-	IsMature             string `json:"is_mature"`
+	IsMature             bool   `json:"is_mature"`
```

Comment:
Does 'bool' work when it comes in as JSON? Will it be 'true' or 'false' anyway? Probably fine either way though.

Reply:
It does. 

```
{"subscription":{"id":"5d693d1d-31ca-6e9a-74da-8ea31a22ffaa","status":"enabled","type":"channel.update","version":"1","condition":{"broadcaster_user_id":"18684204"},"transport":{"method":"webhook","callback":"null"},"created_at":"2021-06-01T16:44:53.158616Z","cost":0},"event":{"broadcaster_user_id":"18684204","broadcaster_user_login":"testBroadcaster","broadcaster_user_name":"testBroadcaster","title":"Example title from the CLI!","language":"en","category_id":"509658","category_name":"Just Chatting","is_mature":false}}
```

as an example- the actual payload is a JSON boolean value it appears, so updated to match.